app/util/Youtube.scala

```diff
@@ -151,28 +149,26 @@ case class YouTubeVideoUpdateApi(config: YouTubeConfig) extends YouTubeBuilder {
     }
   }
 
-  private def setStatusToPrivate(asset: Asset, atomId: String): Unit = {
-    if (asset.platform == Youtube) {
-      YouTubeVideoInfoApi(config).getVideo(asset.id, "snippet,status") match {
-        case Some(video) => {
-          protectAgainstMistakesInDev(video)
+  def setStatusToPrivate(id: String, atomId: String): Unit = {
```

Comment:
Just discussed this and agreed to address it later on as part of making more of a service for asset management